xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
    plan to use them, read the program documentation.
   </para>
 
-  <para>
-<!-- http://doccomments.provo.novell.com/comments/28800 -->
+  <!-\-<para>
+<!-\\- http://doccomments.provo.novell.com/comments/28800 -\\->
    These days file synchronization can also be done with a cloud computing
    solution.
+  </para>-\->
+ </sect1>-->
+ <warning>
+  <title>Risk of Data Loss</title>
+  <para>
+   Before you start managing your data with a synchronization system, you
+   should be well acquainted with the program used and test its functionality.
+   A backup is indispensable for important files.
   </para>
+ </warning>
 
-  <sect2 xml:id="sec.net.sync.soft.cvs">
-   <title>CVS</title><indexterm>
-   <primary>files</primary>
-   <secondary>synchronizing</secondary>
-   <tertiary>CVS</tertiary></indexterm><indexterm>
-   <primary>CVS</primary></indexterm>
-   <para>
-    CVS, which is mostly used for managing program source versions, offers the
-    possibility of keeping copies of the files on multiple computers.
-    Accordingly, it is also suitable for data synchronization. CVS maintains a
-    central repository on the server in which the files and changes to files
-    are saved. Changes that are performed locally are committed to the
-    repository and can be retrieved from other computers by means of an update.
-    Both procedures must be initiated by the user.
-   </para>
-   <para>
-    CVS is very resilient to errors when changes occur on several computers.
-    The changes are merged and (if changes took place in the same lines) a
-    conflict is reported. When a conflict occurs, the database remains in a
-    consistent state. The conflict is only visible for resolution on the client
-    host.
-   </para>
-  </sect2>
-
-  <sect2 xml:id="sec.net.sync.soft.rsync">
-   <title>rsync</title><indexterm>
-   <primary>rsync</primary></indexterm><indexterm>
-   <primary>files</primary>
-   <secondary>synchronizing</secondary>
-   <tertiary>rsync</tertiary></indexterm>
-   <para>
-    When no version control is needed but large directory structures need to be
-    synchronized over slow network connections, the tool rsync offers
+ <sect1 xml:id="sec.net.sync.rsync">
+  <title>File Copying with Rsync</title>
+  <para>
+    When large directory structures need to be
+    synchronized over slow network connections, the tool Rsync offers
     well-developed mechanisms for transmitting only changes within files. This
     not only applies to text files, but also binary files. To detect the
-    differences between files, rsync subdivides the files into blocks and
+    differences between files, Rsync subdivides the files into blocks and
     computes check sums over them.
    </para>
    <para>
     The effort put into the detection of the changes comes at a price. The
```

Comment:
The effort put into the detection of the changes comes at a price. The systems to synchronize should be scaled generously for the usage of rsync. RAM is especially important.

**Detecting changes requires some computing power. So make sure that that machines on both ends have enough resources, including RAM.**